README.md

```diff
@@ -254,22 +298,49 @@ Rspec.desrcibe TShirt do
 end
 ```
 
-This is the best test file yet. Because we are not using `before` nor `let`, we don't need to search around our test examples to figure out what values are in use. We have reduced the cognitive burn of understanding our test code, and thus made them easier to understand and edit/improve as our `TShirt` class evolves. We also quickly understand what are tests do NOT need.
+This is the best test file yet. Because we are not using `before` nor `let`, we don't need to search around our test examples to figure out what values are in use. We have reduced the cognitive burn of understanding our test code, and thus made them easier to understand and edit/improve as our `TShirt` class evolves. We also quickly understand what are tests do NOT need. `before` and `let` can easily lead to the testing [Obscure Test](http://xunitpatterns.com/Obscure%20Test.html), [Fragile Test](http://xunitpatterns.com/Fragile%20Test.html#Fragile%20Fixture), and Slow Tests](http://xunitpatterns.com/Slow%20Tests.html) testing smells. Avoiding `before` and `let` can help us avoid these smells.
```

Comment:
Perhaps this is a good place to mention that specifications and examples are a form of executable documentation? The benefits we see here by not using context-obscuring features of RSpec like `let` and `before` is that the reader need not traverse away from the example to fully understand the context and intent. Perhaps illustrate that the better portion of a programmer-day is spent reading, and understanding code rather than writing it - and specifications are code?

Reply:
PR welcome for an addition with this. I'm just finishing up edits - then I have to prep this for deployment. Thanks for this review!